app/src/androidTest/java/com/monkeyteam/chimpagne/newtests/TestUtils.kt

```diff
@@ -20,36 +21,58 @@ val TEST_EVENTS =
             id = "FIRST_EVENT",
             title = "First event",
             description = "a random description",
-            location = Location("Paris"),
+            location = Location("EPFL", 46.519124, 6.567593),
             public = true,
-            tags = listOf("vegan"),
+            /* DO NOT DELETE OR MODIFY TAGS*/
```

Comment:
I appreciate the insightful comment and suggest that next time you use single line comments with // when the comment is, well, in a single line.